trial-config-service/src/test/resources/collections/environments/PR.postman_environment.json

```diff
@@ -0,0 +1,29 @@
+{
+	"id": "313754ce-656a-4aa2-ab47-b63c6a6f0537",
+	"name": "PR",
+	"values": [
+		{
+			"key": "client-id",
+			"value": "901b377a-a4c0-4a72-b390-294cef5c0c74",
+			"enabled": true
+		},
+		{
+			"key": "client-secret",
+			"value": "wq99fHb3Q9Vg_LfsZ_pNEG6~XR_h8um6u4",
```

Comment:
Is this a real secret?
If so, it's now compromised. 
There should be a different way to obtain secrets

Reply:
@mdp0 any ideas how we can hide this in the postman configuration ?